db/migrate/20201104175215_add_reminder_email_flags_to_virtual_hearings.rb

```diff
@@ -0,0 +1,12 @@
+class AddReminderEmailFlagsToVirtualHearings < Caseflow::Migration
+  def change
+    add_column :virtual_hearings,
+               :appellant_reminder_sent,
```

Comment:
To mimic Rails convention for timestamps such as `created_at` and `updated_at`:
```suggestion
               :appellant_reminder_sent_at,
```